spec/test-data/translations/glotpress-download/expected/values-fr/strings.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Translation-Revision-Date: 2021-03-31 09:54:04+0000
+Plural-Forms: nplurals=2; plural=n > 1;
+Generator: GlotPress/2.4.0-alpha
+Language: fr
+-->
+<resources>
+    <string name="product_attributes_error_saving">Erreur lors de l’enregistrement de vos attributs</string>
+    <string name="shipping_label_payments_cant_edit_warning">Seul le propriétaire du site peut gérer les moyens de paiement des étiquettes d’expédition. Veuillez contacter le propriétaire de la boutique %1$s (%2$s) pour gérer les moyens de paiement.</string>
+    <string name="shipping_label_shipping_method_free_shipping">Livraison gratuite</string>
+    <string name="shipping_label_shipping_method_local_pickup">Point de vente</string>
+    <string name="shipping_label_shipping_method_flat_rate">Forfait</string>
+    <string name="shipping_label_shipping_carrier_flat_fee_banner_message">Le client a payé %1$s (%2$s) pour la livraison</string>
+    <string name="shipping_label_woo_discount_bottomsheet_message" formatted="false">Lors de l’achat d’étiquettes d’expédition avec WooCommerce, vous bénéficiez de 5 à 40 %% de réduction par rapport aux tarifs de la Poste.</string>
+    <string name="shipping_label_woo_discount_bottomsheet_title">Qu’est-ce que la remise WooCommerce  Services ?</string>
+    <string name="shipping_label_payments_saving_dialog_message">Veuillez patienter…</string>
```

Comment:
To test that `...` has been substituted to `…`

Reply:
Likewise, this is self-documented via – and the presence of this case in the fixtures are guaranteed by – the `shared_examples 'substitutions'` in the spec, which ensures that we have at least one fixture containing `...` as input, and ensures that all `expected/*` files have `…` but don't have `...`. That way if we modify the fixtures accidentally in the future and remove those from the examples/fixtures without realizing that those are here on purpose to test those substitution cases, the unit tests will remind us that we do need to keep those cases in our fixtures.